Site/bc_weather_station_dashboard/website/templates/weather.html

```diff
@@ -131,9 +143,10 @@ <h2 class="card-text" id="wind-speed">N/A</h2>
                     </div>
                     <!-- Wind Direction Card -->
                     <div class="col-md-4">
-                        <div class="card rounded shadow mb-md-0 mb-4">
```

Comment:
Removing this got rid of the card styling

Reply:
Added it back.